contracts/src/periphery/cert/interfaces/IEigenDACertTypeBindings.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
+import {EigenDACertTypes} from "src/periphery/cert/EigenDACertTypes.sol";
+
+/// @dev The EigenDA team requires ABIs for EigenDA certificate types.
+///      However, ABIs for types are not generated by the solidity compiler without a function defined.
+///      This interface is simply a workaround for this limitation.
+interface IEigenDACertTypeBindings {
+    function dummyVerifyDACertV1(
+        DATypesV1.BlobHeader calldata blobHeader,
+        DATypesV1.BlobVerificationProof calldata blobVerificationProof
+    ) external view;
+
+    // There is no need for a V2 dummy because the V2 types are available in the V3 cert.
```

Comment:
for backwards compatibility on the clients backend we still need access to the VerifyDACertV2 method for doing historical calls when deriving

Reply:
WDYM change from type to type? I don't think I understand this. I also don't really understand the nomenclature either, since the IEigenDACertBuilder is not just building certs. It is the actual cert verifier that is behind a router.

Reply:
Sorry meant from "cert version to cert version". Perhaps the "CertBuilder" naming can be improved, but the idea is that all of these methods are only to transform the types like BlobInfo received from the backend into a CertBuilder. They are not actually needed to verify a cert.